apps/test-suite/tests/Contacts.js

```diff
@@ -143,6 +146,21 @@ export async function test({ describe, it, xdescribe, jasmine, expect, afterAll
       return createContact(fields);
     }
 
+    async function createContactWithStarred() {
+      const fields = {
+        [Contacts.Fields.Starred]: true,
+        [Contacts.Fields.FirstName]: 'My',
+        [Contacts.Fields.LastName]: 'Best Friend',
+      };
+
+      return createContact(fields);
+    }
+
+    it('Contacts.createContactAsync() with starred', async () => {
+      const contactId = await createContactWithStarred();
+      expect(typeof contactId).toBe('string');
+    });
+
     it('Contacts.createContactAsync() with image', async () => {
```

Comment:
please delete the `Contacts.createContactAsync() with image` test case, it's covered by ` Contacts.getContactByIdAsync() retrieves image`